src/com/projectkorra/projectkorra/ability/FireAbility.java

```diff
@@ -241,6 +241,7 @@ public static void removeFire() {
 	 * 
 	 * @param location The Location
 	 */
+	@SuppressWarnings("deprecation")
```

Comment:
Typically we want to remove warning suppressions, not add them, especially when it comes to depreciation. It is important to see the places where code can potentially break in the future.

Reply:
Understood, I simply saw that there were errors and there were ways to fix them. I'll leave them alone next time.

Reply:
I am not going to accept this current PR unless you address the comments I made throughout it. Please either make the changes or close this request.

Reply:
I would love to remove the unneeded commits but I couldn't find the buttons so I'll close this and make one just for the language.yml